### PR TITLE
Adjusted install command width

### PR DESCRIPTION
**Pull request by pratap0007**

### CODE REVIEW

File: frontend/src/components/basic-detail/BasicDetail.tsx [hunk 1]
```diff
@@ -58,42 +53,36 @@ const BasicDetail: React.FC<BasicDetailProp> = (props: BasicDetailProp) => {
...
-            isReadOnly variant={ClipboardCopyVariant.expansion}>
-            {`kubectl apply -f ${it.it}`}</ClipboardCopy>
-          <br />
-        </FlexItem>
-      </Flex>
+      <React.Fragment>
+        <ClipboardCopy style={{marginBottom: '2em'}}
+          isReadOnly variant={ClipboardCopyVariant.expansion}>
+          {`kubectl apply -f ${it.item}`}</ClipboardCopy>
+      </React.Fragment>
```
Comment (sthaha) [inline frontend/src/components/basic-detail/BasicDetail.tsx:62]: where does the `Flex` get closed?

Comment (pratap0007) [reply frontend/src/components/basic-detail/BasicDetail.tsx]: I have used Flex in return which is closing at line no.  193

Comment (pratap0007) [reply frontend/src/components/basic-detail/BasicDetail.tsx:62]: > where does the `Flex` get closed?

I have used Flex in return which is closing at line no 193

**Review by PuneetPunamiya — COMMENTED** [review]
It is working fine for me. The install command width is now fixed properly.

**Pull request merged**